src/game/client/draw/w3dmodeldraw.cpp

```diff
@@ -2970,7 +2970,6 @@ void W3DModelDraw::Xfer_Snapshot(Xfer *xfer)
                         if (anim) {
                             float frame;
                             int frames;
-                            int mode;

```

Comment:
This looks like an error. `int mode` was shadowing the serialized mode from above.